README.md

```diff
@@ -2,36 +2,50 @@
 
 ## Overview
 
-A brief description. Be clear about the purpose of the resolver,
-its capabilities and limitations.
+A custom filter for Sceptre for unquoting resolvers appearing in var files.
 
 ## Installation
 
 Installation instructions
 
 To install directly from PyPI
 ```shell
-pip install sceptre-resolver-template
+pip install jinja-unquote-resolvers-filter
 ```
 
 To install from the git repo
 ```shell
-pip install git+https://github.com/Sceptre/sceptre-resolver-template.git
+pip install git+https://github.com/Sceptre/jinja-unquote-resolvers-filter.git
 ```
 
 ## Usage/Examples
 
-Refer to the [Sceptre resolver docs](https://docs.sceptre-project.org/latest/docs/resolvers.html#resolvers)
+Refer to the [Sceptre resolver docs](https://docs.sceptre-project.org/latest/docs/filters.html#filters) <!-- This page does not exist yet. -->
```

Comment:
I don't think we are going to add a section in Sceptre docs for this.  Alll docs should be in this project and it should probably reference the Sceptre jinja environment section, https://docs.sceptre-project.org/latest/docs/stack_group_config.html?highlight=jinja#j2-environment